setup.py

```diff
@@ -34,6 +34,7 @@
         "azure-identity",
         "azure-storage-blob>=12.12.0",
         "fsspec>=2021.10.1",
+        "aiohttp>=3.7.0",
```

Comment:
We probably should add a note here or something. Otherwise https://github.com/fsspec/adlfs/pull/335 will happen again.

Reply:
cc @skshetry